salt/utils/event.py

```diff
@@ -328,7 +373,7 @@ def _get_event(self, wait, tag, pending_tags):
 
         return None
 
-    def get_event(self, wait=5, tag='', full=False, use_pending=False, pending_tags=None):
```

Comment:
Same thing here. If we're going to remove a kwarg, let's put it on a deprecation path. This function is heavily used by software even outside of salt's core.